test/e2e/hotGraphic.cy.js

```diff
@@ -16,18 +17,29 @@ describe('Hot Graphic', function () {
 
     // Check pin popup navigation works as expected
     cy.get('.hotgraphic__pin-item .item-0').click();
-    items.forEach(item =>  {
+    items.forEach(item => {
       cy.testContainsOrNotExists('.hotgraphic-popup__item-title-inner', item.title);
       cy.testContainsOrNotExists('.hotgraphic-popup__item-body-inner', item.body);
       cy.get('.hotgraphic-popup__controls.next').click();
-    })
-    cy.get('.hotgraphic-popup__controls.next.is-disabled').should('exist');
-    items.forEach(item =>  {
+    });
+
+    if (!canCycle) {
+      cy.get('.hotgraphic-popup__controls.next.is-disabled').should('exist');
+    } else {
+      cy.get('.hotgraphic-popup__controls.next').should('exist');
```

Comment:
```suggestion
      cy.get('.hotgraphic-popup__controls.next').should('exist');
      cy.get('.hotgraphic-popup__controls.next.is-disabled').should('not.exist');
```

Reply:
Do we need to check if .next.is-disabled does not exist because if .next exists then .next.is-disabled can't?

Reply:
I might be wrong here, but I don't think the existence of a `.hotgraphic-popup__controls.next` excludes the possibility of that element also having the class `is-disabled` applied to it
So by doing 2 Cypress checks we are confirming the expected element exists, but confirming that it does not have the `is-disabled` class

Reply:
^ I think we could probably do it in one command or something to check is the `.hotgraphic-popup__controls.next` exists and doesn't have the `is-disabled` class as well. Not sure exactly on the command there though

Reply:
```suggestion
      cy.get('.hotgraphic-popup__controls.next')
        .should('exist')
        .and('not.have.class', 'is-disabled')
```